src/app/pages/organizations/developers/developer/join.jsx

```diff
@@ -22,10 +24,47 @@ import { useFetchDevelopers, usePutJoinDevelopers } from 'api/developer';
 import { ChplActionBar } from 'components/action-bar';
 import { ChplTextField } from 'components/util';
 import { getAngularService } from 'services/angular-react-helper';
+import ClearIcon from '@material-ui/icons/Clear';
+import AddIcon from '@material-ui/icons/Add';
```

Comment:
These are imports from MUI, so should be up around the other MUI imports. This section is for imports from other places in the CHPL code